tests/system/providers/google/cloud/datafusion/example_datafusion.py

```diff
@@ -36,34 +35,36 @@
     CloudDataFusionStopPipelineOperator,
     CloudDataFusionUpdateInstanceOperator,
 )
-from airflow.providers.google.cloud.operators.gcs import GCSCreateBucketOperator
+from airflow.providers.google.cloud.operators.gcs import GCSCreateBucketOperator, GCSDeleteBucketOperator
```

Comment:
From my perspective that would look better, but it's up to you. Both variants are good for me.
```python
from airflow.providers.google.cloud.operators.gcs import (
    GCSCreateBucketOperator,
    GCSDeleteBucketOperator,
)
```